index.js

```diff
@@ -231,8 +226,6 @@ function CfnLambdaFactory(resourceDefinition) {
       
       console.log('RESPONSE: %j', response);
 
-      var https = require('https');
```

Comment:
How does this work on non-webpack? Won't this throw undefined when using url.parse?

Reply:
(or any https call, it's actually the next line where you deleted the requiring of the url module)

Reply:
I only moved it to the top of the file